io.github.fidele007.akkhara.yaml

```diff
@@ -0,0 +1,82 @@
+  - --device=dri # GPU acceleration if needed
+
+modules:
+  - name: libass
+    # mpv won't build without this
+    cleanup:
+      - /include
+      - /lib/pkgconfig
+    config-opts:
+      - --disable-static
+    sources:
+      - type: archive
+        url: https://github.com/libass/libass/releases/download/0.15.0/libass-0.15.0.tar.gz
+        sha256: 9cbddee5e8c87e43a5fe627a19cd2aa4c36552156eb4edcf6c5a30bd4934fe58
+  - name: mpv
+    buildsystem: simple
+    cleanup:
+      - /include
+      - /lib/pkgconfig
+    build-commands:
+      - python3 waf configure --prefix=/app --enable-libmpv-shared --disable-build-date
+        --disable-manpage-build --disable-alsa --enable-libarchive
+        --disable-lua --disable-javascript --disable-uchardet --disable-drm --disable-dvdnav
+      - python3 waf build
+      - python3 waf install
+    post-install:
+      # save screenshots at ~/Pictures/mpv
+      - echo "screenshot-directory=~/Pictures/mpv" > /app/etc/mpv/mpv.conf
+    sources:
+      - type: archive
+        url: https://github.com/mpv-player/mpv/archive/v0.35.0.tar.gz
+        sha256: dc411c899a64548250c142bf1fa1aa7528f1b4398a24c86b816093999049ec00
+      - type: file
+        url: https://waf.io/waf-2.0.22
+        sha256: 0a09ad26a2cfc69fa26ab871cb558165b60374b5a653ff556a0c6aca63a00df1
+        dest-filename: waf
+  - name: akkhara
+    buildsystem: simple
+    only-arches:
+      - x86_64
+    build-commands:
+      # Copy all built files to Flatpak-based location
+      - mkdir -p /app/akkhara
+      - cp -r * /app/akkhara/
+      - chmod +x /app/akkhara/akkhara
+      - mkdir -p /app/bin
+      - ln -s /app/akkhara/akkhara /app/bin/akkhara
+      # Install the icon
+      - install -Dm644 io.github.fidele007.akkhara.svg /app/share/icons/hicolor/scalable/apps/
+      # Install the desktop file
+      - install -Dm644 io.github.fidele007.akkhara.desktop /app/share/applications/
+      # Install the AppStream metadata file.
+      - install -Dm644 io.github.fidele007.akkhara.metainfo.xml /app/share/metainfo/
+    sources:
+      - type: archive
+        url: https://github.com/fidele007/akkhara-support/releases/download/1.2.0/akkhara_1.2.0_amd64.tar.gz
+        sha256: abb0341fddbd513fca5946ed6ad7dcce28de971a1788f1a249e4323ff408c860
+        strip-components: 0
+      - type: file
+        url: https://github.com/fidele007/akkhara-support/raw/main/flathub_stuff/io.github.fidele007.akkhara.desktop
+        sha256: f8956995aec13d6d778f120c5dff7a68b534eab5b836f8dba600ca8cbef9e3f4
+      - type: file
+        url: https://github.com/fidele007/akkhara-support/raw/main/flathub_stuff/io.github.fidele007.akkhara.svg
+        sha256: 212908b0eb3d173456177e2768fb513116cd792f0eb543d90511c12d9ebe2090
+      - type: file
+        url: https://github.com/fidele007/akkhara-support/raw/main/flathub_stuff/io.github.fidele007.akkhara.metainfo.xml
+        sha256: 739df178d5899e22b09e35b9f4c5afe3aca4b76ca968cc871c606f5f864e189e

```

Comment:
these files should NEVER be referenced from a branch, but always be from a specific revision or tag. Otherwise the build will randomly break. Since they belong to a release your rationale of "I only want binaries in the tarball" is VERY WRONG.

You seem to be willing to ignore advice given.

Reply:
and NONE of these 3 files are specific to flathub. Any distro packaging would want them.